test_coordinates.py

```diff
@@ -1,21 +1,25 @@
 from coordinates import Coordinates
 
 # basic_science
+basic_science = Coordinates("basic_science", [12.96626, 77.71211])
```

Comment:
added coordinates of nodes (basic_science, ganesha_statue, ganesha_statue, ganesha_statue)